custom_types.yaml

```diff
@@ -587,12 +587,12 @@ node_types:
     interfaces:
       Standard:
         configure:
-          implementation: mesos/mesos_master_install.yml
+          implementation: artifacts/mesos/mesos_master_install.yml
           inputs:
-            consul_server_ips: { get_attribute: [ HOST, public_address ] }
-            zookeeper_host_list_ips: { get_attribute: [ HOST, public_address ] }
-            zookeeper_peers_ips: { get_attribute: [ HOST, public_address ] }
-            mesos_masters_list_ips: { get_attribute: [ HOST, public_address ] }
+            consul_server_ips: [ { get_attribute: [ HOST, public_address ] } ]
+            zookeeper_host_list_ips: [ { get_attribute: [ HOST, public_address ] } ]
+            zookeeper_peers_ips: [ { get_attribute: [ HOST, public_address ] } ]
+            mesos_masters_list_ips: [ { get_attribute: [ HOST, public_address ] } ]
```

Comment:
hum that's a problem for me. The spec already defines public_address on tosca.nodes.Compute as the IP of the node, so the translator uses this fact to replace it with an equivalent HOT function. Can we rename them to *_addresses and/or move them under the scalable capability ?